File: Array/Cost of Groceries.py
testcase = int(input())

for i in range(testcase):
    no_of_item, minimumfreshnessvalue = input().split()
    no_of_item, minimumfreshnessvalue = int(no_of_item), int(minimumfreshnessvalue)
    freshnessvalue = list(map(int, input().split()))
    costofitem = list(map(int, input().split()))

    sum = 0
    for i in range(len(freshnessvalue)):
        if freshnessvalue[i]>=minimumfreshnessvalue:
            sum+=costofitem[i]
    print(sum)

File: Array/Cost of Groceries.cpp
#include<iostream>
#include<vector>
using namespace std;

int main()
{
    int testcase;
    cin>>testcase;

    while (testcase--)
    {
        int no_of_item, minfreshnessvalue;
        vector<int> freshnessvalue(no_of_item);
        vector<int> costofitem(no_of_item);

        for (int i = 0; i < no_of_item; i++)
        {
            cin>>freshnessvalue[i];
        }

        for (int i = 0; i < no_of_item; i++)
        {
            cin>>costofitem[i];
        }

        int sum=0;
        for (int i = 0; i < no_of_item; i++)
        {
            if (freshnessvalue[i]>=minfreshnessvalue)
            {
                sum+=freshnessvalue[i];
            }
        }
        
        cout<<sum<<endl;
        
    }
}